### PR TITLE
PLAT-104238: Support accessibility for Picker components

### PR DESCRIPTION
**Pull request by Baekwoo-s**
Support accessibility for Picker components

Enact-DCO-1.0-Signed-off-by: Baekwoo Jung (baekwoo.jung@lge.com)

### CODE REVIEW

File: Picker/Picker.js [hunk 1]
```diff
@@ -63,6 +65,53 @@ const PickerBase = kind({
...
 
+		/**
+		 * Prevents read out of both the slider and the increment and decrement
+		 * buttons.
+		 *
+		 * @type {Boolean}
+		 * @memberof agate/Picker.PickerBase.prototype
+		 * @public
+		 */
+		'aria-hidden': PropTypes.bool,
```
Comment (seunghoh) [inline Picker/Picker.js:76]: Please check if we really need to define as a prop. This is the same as dom attribute and react can pass to the dom

File: Picker/Picker.js [hunk 2]
```diff
@@ -116,12 +176,20 @@ const PickerBase = kind({
 						{isFirst ? '' : values[value - 1]}
 					</div>
 				</PickerButtonItem>
-				<div className={activeClassName}>
+				<div
+					aria-valuetext={typeof valueText !== 'undefined' ? valueText : values[value]}
+					className={activeClassName}
+					id={id}
```
Comment (ryanjduffy) [inline Picker/Picker.js:182]: Let's keep `id` on the root and have a derived id here (e.g. `${id}_value`) that is used by `aria-controls`. You might also want to create it in computed and only deconstruct it in render so the id lands on the root automatically.

Comment (Baekwoo-s) [reply Picker/Picker.js:183]: I fixed the id to be modified in computed.
Please tell me if this code is different from your intention.

Comment (ryanjduffy) [reply Picker/Picker.js:183]: Slightly, yes. I meant something like:

```
computed: {
  valueId: ({id}) => `${id}_value`
}
```

And then only deconstruct `valueId` from `props` so `id` lands on the root node and you use `valueId` for `aria-controls` and this node's id.

Comment (Baekwoo-s) [reply Picker/Picker.js]: I changed the name of the id prop according to your opinion.
Thank you.

**Pull request merged**